### DISCUSSION TITLE
SyntaxError: Parser error in on.js (Safari 15.5)

### DISCUSSION THREAD

**Discussion by lessless** [alpinejs/alpine#2965] [7. Bugs (archived)]
Hi,

I can see the following error in the Safari 15.5

```
[Error] SyntaxError: Parser error
	on (app.js:8735)
	(anonymous function) (app.js:9267)
	(anonymous function) (app.js:7615)
	(anonymous function)
	flushHandlers (app.js:7029)
	stopDeferring (app.js:7034)
	deferHandlingDirectives (app.js:7037)
	initTree (app.js:7237)
	(anonymous function) (app.js:7197)
	forEach
	start (app.js:7196:95)
	setupAlpine (app.js:9314)
	handleDOMContentLoaded (app.js:9309)
```

The `app.js:8735` (top item in the backtrace) points to [line 2345 in module.esm.js](https://unpkg.com/browse/alpinejs@3.10.2/dist/module.esm.js), which is probably https://github.com/alpinejs/alpine/blob/main/packages/alpinejs/src/utils/on.js#L52

Best,
Yevhenii

**Comment by SimoTod** [comment]
This usually means that there's a typo in your directive (what you wrote in the html) after the error in console, alpine logs a warning showing you the element that triggered the error.

**Comment by lessless** [comment]
Got ya, thank you!
